src/napari/layers/intensity_mixin.py

```diff
@@ -63,7 +63,7 @@ def reset_contrast_limits(self: 'ScalarFieldBase', mode=None):
     def _calc_data_range(self, mode):
         raise NotImplementedError
 
-    def reset_contrast_limits_range(self, mode=None):
+    def reset_contrast_limits_range(self, mode=None) -> None:
         """Scale contrast limits range to data type if dtype is an integer,
         or use the current maximum data range otherwise.
         """
```

Comment:
I do not understand why mypy does not complain here. 
```suggestion
    def reset_contrast_limits_range(self, mode: Literal['data', 'slice'] | None = None) -> None:
        """Scale contrast limits range to data type if dtype is an integer,
        or use the current maximum data range otherwise.
        
        Parameters
        ----------
        mode: 'data' or 'slice'
           If range should be calculated on whole data, or only visible slice. 
        """
```